procedures/foreman/testweek.md.erb

```diff
@@ -0,0 +1,100 @@
+Foreman <%= full_version %> has been announced. **We need your help!** It's easy to get your hands dirty and help us with testing:
+
+1. Install Foreman RC version (the most up to date one)
+1. Pick a scenario from this post or add your own scenario
+1. If you find an issue, file it in the RedMine and **make a comment in this thread** linking the issue
+1. Mark the scenario as checked in this OP (this is an editable wiki post), here is syntax of checked and unchecked lines (you can click on checkboxes directly with mouse):
+   - [ ] Unchecked
+   - [x] Checked
+
+**Installation**
+* Install Foreman using existing script/forklift/beaker
+  * [ ] Forklift
+  * [ ] My own install script
+* Install Foreman *manually* by following our [installation guide](https://theforeman.org/manuals/latest/index.html)
+  * [ ] RHEL / CentOS stable
+  * [ ] Debian / Ubuntu stable
+* Install Foreman *manually* by following our new [installation guide](https://docs.theforeman.org)
+  * [ ] RHEL / CentOS
+  * [ ] Debian / Ubuntu
+* Upgrade existing Foreman deployment (advertise in RedMine it was an upgraded instance if you encounter bug)
```

Comment:
Not always however sometimes we have upgrade notes, I think its good to link it here? so users know what may not work etc to avoid false issues on redmine?

Reply:
We could link to https://theforeman.org/manuals/3.1/index.html#3.6Upgradeto3.1 (s/3.1/$release/). Perhaps that should say "be sure to have read $upgrade_instructions"? I'd like to keep the instructions here minimal.